### ISSUE TITLE
превод на flag

### ISSUE THREAD

**Issue by stelf** [stelf/en2bg4term#25]
дали байрак е адекватен превеод на flag?

**Comment by metala** [comment]
-1 ако ще е нещо такова, поне да ползваме българската дума 'знаме / *знаменце*'.

За мен думата има няколко значения, започвайки с това че е едновременно глагол и съществително. 

При едното има знаменце, което указва случило се събитие, напр. Overflow flag, flag register. А другото е тип 'flag inappropriate content'.

**Comment by stelf** [comment]
знаме може би няма да се възприеме, иначе е за мен важи. други предложения?

**Comment by vlood** [comment]
Виждайки дискусията за сензорния екран ми се струва излишно, но все пак ще предложа "маркер" и "маркиране"/"отбелязване" за съществителното, респективно за глагола.

**Comment by metala** [comment]
'отбелязване', звучи добре, но обикновено изисква допълнителен контекст. За маркер, бих казал, че е добре, дума е синоним за едно от тези, в зависимост от контекста: отметка, показалец, знак, белег, обозначение, означение
.

**Comment by stelf** [comment]
знак звучи добре, малко двусмислено с character. 

показалец ми харесва, единствено риск, че винаги ги критикуват по-дългите думи, а в английския има доста кратки.

**Comment by hkdobrev** [comment]
Малко контекст:

Според мен наложилото се flag в техническите термини идва от флаговете на пощенските кутии в САЩ, които имат вдигнато или свалено положение, за да отрази дали има поща или няма поща. От там се е пренесло в email клиентите и съответно се използва и за всякакви други boolean "флагове".

Други знамена и флагове, нямат двоичен смисъл както на пощенските кутии.

Предлагам, преводът доколкото е възможно да означава, че има флагът има двоична стойност (знам, че има такива и с повече стойности, но там обикновено се указва изрично).

---

На български съответно знаме или байрак няма такъв контекст. Маркер е по-близко, но има други видове маркери в смисъла на курсор или указател.

Според мен на български близка дума е ["семафор"](https://bg.m.wikipedia.org/wiki/%D0%A1%D0%B5%D0%BC%D0%B0%D1%84%D0%BE%D1%80).

> Семафорът (от гр. – σήμα (знак) + φορός (носещ)) е устройство, показващо на влаковите машинисти **дали** железопътната линия пред тях **е заета или свободна**, дали ще се премине по главния коловоз, или през отклонението и др. Семафорите биват механически и светлинни, като най-често думата „семафор“ се използва за механическите, а светлинните се наричат светофори.

**Comment by hkdobrev** [comment]
В програмирането семафор има малко по-широк смисъл, но също може да се използва като toggle.

https://en.m.wikipedia.org/wiki/Semaphore_(programming)

**Comment by sfbg** [comment]
Да не забравяме, че _флаг_ си е и утвърдена българска дума.
Не е като _комитвам_, примерно.

Не мисля, че трабва да губим енергия за точно такива.

**Comment by hkdobrev** [comment]
Да, но флаг на български има значение само в смисъл близък до знаме.

**Comment by stelf** [comment]
окей, предложение

знак, белег, 

и защото, когато седя на единия хълм и на другия някой вдигне знаменцето, ми дава знак. прави ми знак. а ако сложа знаменце (😅 знамение! 😆) на някое писмо, де факто му слагам белег (или нишан както казват, което е турска или персиска, кой знае).

бтв, оня ден слушах един езиковед по Христо Ботев на БНР и той сподели как хляб всъщност е чуждица, ама толкова сме свикнали, че няма да ни направи значение. така че може и спокойно да допишем флаг, но наистина, както отбелязва @hkdobrev, на български съвсем друго значи ... а именно - знаме.

->@metala / @vlood

**Comment by vlood** [comment]
„знак“ е най-близо до търсеното значение, според мен. Надали бих го ползвал, но ако флаг не е ок, следващото за мен е „знак“.
